### ISSUE TITLE
Figure 2.3 intersects Example 2.2 box

### ISSUE THREAD

**Issue by Pugio** [mml-book/mml-book.github.io#459]
**Describe the mistake**
The gray box from Example 2.2 flows onto the next page, where it is interrupted by Figure 2.3. This causes the reader to think that Fig 2.3 is a part of Ex 2.2's discussion, when it is actually a part of the text _after_ Ex 2.2 .

**Location**
Version 2019-12-11
Chapter 2, Page 20
Example 2.2 and Figure 2.3

**Proposed solution**
Render Fig 2.3 _after_ the Ex 2.2 box has finished.

**Comment by mpd37** [comment]
Agree, this is suboptimal, but we can't change the layout at this point. Normally, figures, which are part of the example, should be within the gray box as well.
